packages/theme-compiler/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "type": "module",
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=14.0.0"
```

Comment:
Why downgrade the minimum requirement?

Reply:
SVGO only requires node 14+.

Reply:
Reverted back to 18 promoting usage of active LTS.